app/views/declaration/destinationCountries/origination_country.scala.html

```diff
@@ -46,7 +46,7 @@
         @helper.CSRF.formField
 
         @components.fields.field_autocomplete(
-            form("country"),
+            form("code"),
```

Comment:
I would have been explicit and name the variable as `countryCode`, but that looks fine as well

Reply:
I thought about the same, but I prefer to stay with it right now, to not block Maciej.

I'll prepare PR with this change today.